lua/hop/init.lua

```diff
@@ -45,6 +45,8 @@ local function hint_with(mode, opts)
 
   local winblend = opts and opts.winblend or defaults.winblend
 
+  local save_virtualedit = vim.api.nvim_get_option('virtualedit')
+  vim.api.nvim_set_option('virtualedit', 'all')
```

Comment:
I think this caused a regression, as now the buffer is in `virtualedit` mode even though it wasn’t.

Reply:
But we restore this option after opening hop window 
```vim
vim.api.nvim_set_option('virtualedit', save_virtualedit)
```

Reply:
I think this is not the right way to do it? I reproduce every time:

1. Open any buffer.
2. Call any Hop commands.
3. When Hop quits, the window is now in `virtualedit`.

Do you remember why we needed that option? I don’t recall the need. :confused:

Reply:
https://github.com/phaazon/hop.nvim/issues/12